pandas/io/meson.build

```diff
@@ -0,0 +1,36 @@
+subdirs_list = [
+    # exclude sas, since it contains extension modules
+    # and has its own meson.build
+    'clipboard',
+    'excel',
+    'formats',
+    'json',
+    'parsers'
+]
+foreach subdir: subdirs_list
+    install_subdir(subdir, install_dir: py.get_install_dir(pure: false) / 'pandas/io')
+endforeach
+top_level_py_list = [
+    '__init__.py',
+    '_util.py',
+    'api.py',
+    'clipboards.py',
+    'common.py',
+    'feather_format.py',
+    'gbq.py',
+    'html.py',
+    'orc.py',
+    'parquet.py',
+    'pickle.py',
+    'pytables.py',
+    'spss.py',
+    'sql.py',
+    'stata.py',
+    'xml.py'
+]
+foreach file: top_level_py_list
```

Comment:
Why do we need to do this for python modules?

Reply:
The ``io`` directory can't be installed with ``install_subdir`` unfortunately since the sas extension modules live there (and we need to have custom logic for that).

So we have to install each of the top-level files under io separately (since we are not using ``install_subdir``). The rest of the files in folders are fine, since they can be installed with ``install_subdir``.

Ideally, this file disappears when the sas Cython files are moved to _libs. I can do that in a follow-up if you'd like?

Reply:
Instead of a for loop you could probably just pass the entire array as a variadic argument to py.install_sources